subjects/java/checkpoints/distinct-substring-lenght/README.md

```diff
@@ -0,0 +1,45 @@
+## Distinct Substring length
+
+### Instructions
+
+Create a class `DistinctSubstringLength` that provides a method to find the length of the longest substring without repeating characters in a given string.
```

Comment:
What if a string is made of all unique char? Is the length of the "longest substring without repeating characters" supposed to be the length of the string?

Reply:
yes~